docs/installation/windows.md

```diff
@@ -11,8 +11,13 @@ There is a shortcut placed in the Start Menu upon installation.
 
 * By default the package is installed to `$env:ProgramFiles\PowerShell\`
 * You can launch PowerShell via the Start Menu or `$env:ProgramFiles\PowerShell\powershell.exe`
-* Note: On Windows 8.1 / Windows 2012R2, ensure Visual C++ Redistributable
-    for VS2015 is installed from [here](http://download.microsoft.com/download/9/3/F/93FCF1E7-E6A4-478B-96E7-D4B285925B00/vc_redist.x64.exe).
+
+### Prerequisites
+
+* Install the [Universal C Runtime](https://www.microsoft.com/en-us/download/details.aspx?id=50410).
```

Comment:
This seems to indicate to me that the Universal C Runtime is needed on all platforms event Windows 10.